bin/sender.py

```diff
@@ -120,9 +120,18 @@ def main():
                 raise Ssm2Exception('No destination queue is configured.')
         except ConfigParser.NoOptionError, e:
             raise Ssm2Exception(e)
-    
+
+        # Determine what type of message store we are interacting with,
+        # i.e. a dirq QueueSimple object or a plain MessageDirectory directory.
+        try:
+            path_type = cp.get('messaging', 'path_type')
```

Comment:
What happens if the "messaging" section isn't defined?

Reply:
See lines below - we default to using `dirq` (which is the existing type).